api/institutions/serializers.py

```diff
@@ -306,3 +307,43 @@ def get_absolute_url(self, obj):
                 'version': 'v2',
             },
         )
+
+
+class NewInstitutionUserMetricsSerializer(JSONAPISerializer):
```

Comment:
Obviously calling something `New`/`Old` is kind of silly considering it's not versioned in anyway, but hey go for it. At least it's clear what we are doing with this naming convention, it can always be renamed to `SecondNewest` when it's no longer new in 2034. :)

Reply:
really hoping we manage to get around to deleting code at some point...